### PR TITLE
Update Chinese document (guide) to synchronize with nuxt2.1 English document

### PR DESCRIPTION
**Pull request by warriorBrian**
All the documents in the guide directory have been updated, new pages are missing, and so on.
Subsequent updates will continue to update the documentation in the **API directory** and **other Chinese documents :)**

### CODE REVIEW

**Comment by warriorBrian** [comment]
@clarkdo Whether to forget this PR?

**Comment by clarkdo** [comment]
@warriorBrian 感谢您的翻译! 😸

**Comment by warriorBrian** [comment]
@clarkdo 不客气。也谢谢您的肯定。:smile:

File: zh/guide/menu.json [hunk 1]
```diff
@@ -78,6 +78,15 @@
           { "to": "#只在浏览器里使用的插件", "name": "只在浏览器里使用的插件" }
         ]
       },
+      {
+        "to": "/modules", "name": "模 块",
```
Comment (warriorBrian) [inline zh/guide/menu.json]: 我添加了缺失 “/modules” 目录并增加了新的页面，我查看合并后的官网似乎并没有生成。本地跑不起来docs，能否解决这个问题？
@clarkdo  😺

Comment (warriorBrian) [inline zh/guide/menu.json]: 我添加了缺失 “/modules” 目录并增加了新的页面，我查看合并后的官网似乎并没有生成。本地跑不起来docs，能否解决这个问题？
@clarkdo  😺

**Comment by warriorBrian** [comment]
我添加了缺失 “/modules” 目录并增加了新的页面，我查看合并后的官网似乎并没有生成。本地跑不起来docs，是否能解决这个问题？
@clarkdo  😺

**Comment by clarkdo** [comment]
@Atinux 

@warriorBrian added the new menu `Module` in Chinese version doc, but it seemed not displayed in site, could you restart or redeploy now service ?

**Comment by warriorBrian** [comment]
@clarkdo 
不能，我拉取了`docs`的仓库，运行 `npm run dev`. 它监听 **port :4000**, 但是我没有任何返回，只是提示 file is not find. 我没办法定位 `/modules` 不显示问题. 只能来找您解决这个问题。

**Comment by warriorBrian** [comment]
@clarkdo 
I have no way to locate the problem that is not displayed, because the docs repository can't run normally, the console can only see the monitor port 4000, the web page reports an error, so I can only modify the document rendering against the English document, so I can only find you to solve it. Modules` question.

**Comment by clarkdo** [comment]
抱歉，我可能表述有误，我上一条留言是给Atinux的，请他重新部署一下文档的系统😄

**Comment by warriorBrian** [comment]
@clarkdo 感谢您的持续关注！ 😸

File: zh/guide/assets.md [hunk 2]
```diff
@@ -18,19 +18,21 @@ description: 默认情况下 Nuxt 使用 vue-loader、file-loader 以及 url-loa
...
-如果我们在CSS代码中使用 `url('~assets/image.png')`, 那么编译后它将被转换成 `require('~assets/image.png')`。
+如果我们在CSS代码中使用 `url('~assets/image.png')`, 那么编译后它将被转换成 `require('~/assets/image.png')`。
+
+> 请注意，由于css-loader升级，从Nuxt 2.0开始，你应该在CSS文件中使用`~assets`（没有斜杠），例如background:url("~assets/banner.svg")
 
 又或者如果我们在 `pages/index.vue` 中使用以下代码引用图片资源：
 ```html
 <template>
-  <img src="~assets/image.png">
+  <img src="~/assets/image.png">
```
Comment (warriorBrian) [inline zh/guide/assets.md]: **i added `~/assets` But the Chinese official website shows that it is still wrong.**
@Atinux 
tim 20181018115213

**Comment by warriorBrian** [comment]
### The Chinese official website also has the following problems:
#### Go to the assets directory：

qd 7wxudxqv3u jj3qa9 j

### When I refreshed：
####  When refreshed, it returns to the wrong page instead of the updated page:

tim 20181018120900

@Atinux  @clarkdo  😸

**Pull request merged**